src/registrar/tests/test_models_domain.py

```diff
@@ -1,54 +1,487 @@
+        """
+        # implementation note: this requires seeing what happens when these are actually
+        # sent like this, and then implementing appropriate mocks for any errors the
+        # registry normally sends in this case
+        raise
+
+    @skip("not implemented yet")
+    def test_user_deletes_security_email(self):
+        """
+        Scenario: Registrant clears out an existing security contact email
+            Given a domain exists in the registry with a user-added security email
+            When `domain.security_contact` is set equal to a PublicContact with an empty
+                security contact email
+            Then Domain sends `commands.UpdateDomain` and `commands.DeleteContact`
+                to the registry
+            And the domain has a valid security contact with CISA defaults
+            And disclose flags are set to keep the email address hidden
+        """
+        raise
+
+    @skip("not implemented yet")
+    def test_updates_security_email(self):
+        """
+        Scenario: Registrant replaces one valid security contact email with another
+            Given a domain exists in the registry with a user-added security email
+            When `domain.security_contact` is set equal to a PublicContact with a new
+                security contact email
+            Then Domain sends `commands.UpdateContact` to the registry
+        """
+        raise
+
+    @skip("not implemented yet")
+    def test_update_is_unsuccessful(self):
+        """
+        Scenario: An update to the security contact is unsuccessful
+            When an error is returned from epplibwrapper
+            Then a user-friendly error message is returned for displaying on the web
+        """
+        raise
+
+
+class TestRegistrantNameservers(TestCase):
+    """Rule: Registrants may modify their nameservers"""
+
+    def setUp(self):
+        """
+        Background:
+            Given the registrant is logged in
+            And the registrant is the admin on a domain
+        """
+        pass
+
+    @skip("not implemented yet")
+    def test_user_adds_one_nameserver(self):
+        """
+        Scenario: Registrant adds a single nameserver
+            Given the domain has zero nameservers
+            When `domain.nameservers` is set to an array of length 1
+            Then `commands.CreateHost` and `commands.UpdateDomain` is sent
+                to the registry
+            And `domain.is_active` returns False
+        """
+        raise
+
+    @skip("not implemented yet")
+    def test_user_adds_two_nameservers(self):
+        """
+        Scenario: Registrant adds 2 or more nameservers, thereby activating the domain
+            Given the domain has zero nameservers
+            When `domain.nameservers` is set to an array of length 2
+            Then `commands.CreateHost` and `commands.UpdateDomain` is sent
+                to the registry
+            And `domain.is_active` returns True
```

Comment:
No, it is unrelated. The intent is more to underline the fact that once 2 valid nameservers are set, the domain should be live. This could be tested here, but may also be better split into its own test of `is_active`.